app/controllers/supervisor/base_controller.rb

```diff
@@ -0,0 +1,18 @@
+class Supervisor::BaseController < ApplicationController
```

Comment:
It's a taste thing. Personally, I find the explicit module too verbose for a controller (and for multiple nested modules). In cases like that, I would stick with what the project already started out with.

Since we've now mixed both versions, I leave it up to you, @F3PiX, to decide ;)